logic/calendar.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+from abc import ABCMeta, abstractmethod
+from dataclasses import dataclass
+from os import getenv
+
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+
+
+@dataclass
+class Event:
+    title: str
+    date: str
+    starttime: str
+    endtime: str
+    location: str
+
+    def _get_datetime(date, time) -> str:
+        today = datetime.datetime.today()
+        splitteddate = date.split("/")
+        splittedtime = time.split(":")
+
+        return datetime.datetime(
+            today.year,
+            int(splitteddate[0]),
+            int(splitteddate[1]),
+            int(splittedtime[0]),
+            int(splittedtime[1]),
+        ).isoformat()
+
+    @property
+    def get_starttime(self) -> str:
+        datetime = Event._get_datetime(self.date, self.starttime)
+
+        return datetime
+
+    @property
+    def get_endtime(self) -> str:
+        datetime = Event._get_datetime(self.date, self.endtime)
+
+        return datetime
+
+
+class ICalendar(metaclass=ABCMeta):
+    @abstractmethod
+    def _get_service():
+        pass
+
+    @abstractmethod
+    def _create_event_obj(newevent: Event) -> bytes:
+        pass
+
+    @classmethod
+    @abstractmethod
+    def add_event(cls, newevent: Event) -> None:
+        pass
+
+
+class GoogleCalendarImpl(ICalendar):
+    def _get_service():
```

Comment:
```suggestion
    @staticmethod
    def _get_service():
```
引数取らないなら `@staticmethod` じゃない？